### PR TITLE
rebase: update main.go, go mod files and docs (3/8)

### PR DESCRIPTION
**Pull request by jaideepr97**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

> /kind bug
> /kind chore
> /kind cleanup
> /kind failing-test
> /kind enhancement
> /kind documentation
> /kind code-refactoring

**What does this PR do / why we need it**:
3/8 PRs to rebase the redesgin branch

This PR does:
- update main.go 
- update go.mod and go.sum
- update docs folder 

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: docs/install/manual.md [hunk 1]
```diff
@@ -50,9 +50,15 @@ metadata:
     cert-manager.io/inject-ca-from: $(CERTIFICATE_NAMESPACE)/$(CERTIFICATE_NAME)
 ```
 
+<<<<<<< HEAD
```
Comment (ciiay) [inline docs/install/manual.md:53]: Some rebase issue.

File: docs/install/manual.md [hunk 2]
```diff
@@ -50,9 +50,15 @@ metadata:
...
 Enable `../certmanager` directory under the `bases` section in `config/default/kustomization.yaml` file.
 ```yaml
 bases:
+=======
+Enable `../certmanager` directory under the `resources` section in `config/default/kustomization.yaml` file.
+```yaml
+resources:
+>>>>>>> aa399a55abf577809d487fdc2811219cebb1cb35
```
Comment (ciiay) [inline docs/install/manual.md:61]: Rebase issue.

File: docs/reference/argocd.md [hunk 3]
```diff
@@ -137,7 +137,10 @@ spec:
     SCMRootCAConfigMap: example-gitlab-scm-tls-cert
 ```
 
+<<<<<<< HEAD
```
Comment (ciiay) [inline docs/reference/argocd.md:140]: Here too

File: docs/reference/argocd.md [hunk 4]
```diff
@@ -137,7 +137,10 @@ spec:
...
 
+=======
+>>>>>>> aa399a55abf577809d487fdc2811219cebb1cb35
```
Comment (ciiay) [inline docs/reference/argocd.md:143]: Here

File: docs/reference/argocd.md [hunk 5]
```diff
@@ -965,7 +967,11 @@ spec:
 
 Resource behavior can be customized using subkeys (`resourceHealthChecks`, `resourceIgnoreDifferences`, and `resourceActions`). Each of the subkeys maps directly to their own field in the `argocd-cm`. `resourceHealthChecks` will map to `resource.customizations.health`, `resourceIgnoreDifferences` to `resource.customizations.ignoreDifferences`, and `resourceActions` to `resource.customizations.actions`.
 
+<<<<<<< HEAD
```
Comment (ciiay) [inline docs/reference/argocd.md:970]: Here

File: docs/reference/argocd.md [hunk 6]
```diff
@@ -965,7 +967,11 @@ spec:
...
 !!! note 
+=======
+!!! note
+>>>>>>> aa399a55abf577809d487fdc2811219cebb1cb35
```
Comment (ciiay) [inline docs/reference/argocd.md:974]: Here

**Review by ciiay — CHANGES_REQUESTED** [review]
There are a few rebase issues left in the code. The rest looks good.

**Review by ciiay — APPROVED** [review]
LGTM
Thank you!

**Review by svghadi — APPROVED** [review]
/lgtm

**Pull request merged**